kvdb/etcd/embed.go

```diff
@@ -60,8 +60,8 @@ func getFreePort() int {
 // NewEmbeddedEtcdInstance creates an embedded etcd instance for testing,
 // listening on random open ports. Returns the backend config and a cleanup
 // func that will stop the etcd instance.
-func NewEmbeddedEtcdInstance(path string, clientPort, peerPort uint16) (
-	*Config, func(), error) {
+func NewEmbeddedEtcdInstance(path string, clientPort, peerPort uint16,
+	logFile *string) (*Config, func(), error) {
```

Comment:
Just use empty string instead of pointer?

Reply:
done